packages/react-reconciler/src/__tests__/ReactUpdaters-test.internal.js

```diff
@@ -370,14 +433,20 @@ describe('updaters', () => {
     });
     assertLog(['initial', 'onCommitRoot']);
     expect(triggerError).not.toBeNull();
+    expect(allSchedulerTags).toEqual([[HostRoot]]);
+    expect(allSchedulerTypes).toEqual([[null]]);
 
-    allSchedulerTypes.splice(0);
+    allSchedulerTags.length = 0;
+    allSchedulerTypes.length = 0;
     onCommitRootShouldYield = true;
 
     await act(() => {
       triggerError();
     });
+    // TODO: Why the double commit? Isn't this a single commit?
     assertLog(['onCommitRoot', 'error', 'onCommitRoot']);
+    // TODO: If this was a single commit, ErrorBoundary shouldn't be in here.
+    // We only set state in Parent so that should be the only updater
     expect(allSchedulerTypes).toEqual([[Parent], [ErrorBoundary]]);
```

Comment:
`ErrorBoundary` does make sense I believe. It's kind of like a rerender triggered from a child. Though just like the Fiber isn't generally sufficient information for `setState` updates, the error boundary isn't generally sufficient for "rerenders" due to errors. You'd want the boundary + the origin of the error.